### PR TITLE
#VP-6119 migrate to v3

### PR DESCRIPTION
**Pull request by kostyrin**
please note that have added some refactoring for optimization query at WebHookService line 60 .

### CODE REVIEW

File: src/VirtoCommerce.WebHooksModule.Core/Services/IWebhookService.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using System.Threading.Tasks;
+using VirtoCommerce.WebHooksModule.Core.Models;
+
+namespace VirtoCommerce.WebHooksModule.Core.Services
+{
+	public interface IWebHookService
+	{
+		Task<WebHook[]> GetByIdsAsync(string[] ids, string responseGroup = null);
+        Task DeleteByIdsAsync(string[] ids);
```
Comment (yecli) [inline src/VirtoCommerce.WebHooksModule.Core/Services/IWebhookService.cs:9]: Strange indents

File: src/VirtoCommerce.WebHooksModule.Data/Migrations/20201201101415_InitialWebHook.cs [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_WebHookEvent", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_WebHookEvent_WebHook_WebHookId",
+                        column: x => x.WebHookId,
+                        principalTable: "WebHook",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Restrict);
```
Comment (yecli) [inline src/VirtoCommerce.WebHooksModule.Data/Migrations/20201201101415_InitialWebHook.cs]: Maybe it makes sense to add migration to FK to be Cascade

File: src/VirtoCommerce.WebHooksModule.Data/Services/WebhookService.cs [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+                ClearCache();
+            }
+        }
+
+
+        protected virtual void ClearCache()
+        {
+            WebhookCacheRegion.ExpireRegion();
+            WebhookCacheRegion.ExpireRegion();
```
Comment (yecli) [inline src/VirtoCommerce.WebHooksModule.Data/Services/WebhookService.cs]: Duplicated line. Or should it be specific SearchCache region?

File: src/VirtoCommerce.WebHooksModule.Web/Module.cs [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        public ManifestModuleInfo ModuleInfo { get; set; }
+
+        public void Initialize(IServiceCollection serviceCollection)
+        {
+            serviceCollection.AddTransient<IWebHookRepository, WebHookRepository>();
+            serviceCollection.AddDbContext<WebhookDbContext>((provider, options) =>
+            {
+                var configuration = provider.GetRequiredService<IConfiguration>();
+                options.UseSqlServer(configuration.GetConnectionString("VirtoCommerce.Catalog") ?? configuration.GetConnectionString("VirtoCommerce"));
```
Comment (yecli) [inline src/VirtoCommerce.WebHooksModule.Web/Module.cs]: configuration.GetConnectionString("VirtoCommerce.Catalog") ??
is it needed?

File: src/VirtoCommerce.WebHooksModule.Web/module.manifest [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<module>
+    <id>VirtoCommerce.WebHooks</id>
+    <version>3.0.0</version>
+    <version-tag />
+    <platformVersion>3.0.0</platformVersion>
```
Comment (yecli) [inline src/VirtoCommerce.WebHooksModule.Web/module.manifest]: 3.30.0?

File: src/VirtoCommerce.WebHooksModule.Data/Services/WebhookService.cs (hunk 3 above)
Comment (kostyrin) [reply src/VirtoCommerce.WebHooksModule.Data/Services/WebhookService.cs]: Sure, should be SearchCache

File: src/VirtoCommerce.WebHooksModule.Web/Module.cs (hunk 4 above)
Comment (kostyrin) [reply src/VirtoCommerce.WebHooksModule.Web/Module.cs]: fixed

**Review by yecli — DISMISSED** [review]

**Review by yecli — APPROVED** [review]

**Review by akak1977 — APPROVED** [review]

**Pull request merged**